static/js/src/chart-data.js

```diff
@@ -83,18 +83,36 @@ export var serverAndDesktopReleases = [
     taskName: "22.04 LTS (Jammy Jellyfish)",
     status: "ESM",
   },
-  {
-    startDate: new Date("2023-04-20T00:00:00"),
-    endDate: new Date("2024-01-20T00:00:00"),
-    taskName: "23.04 (Lunar Lobster)",
-    status: "INTERIM_RELEASE",
-  },
+  // {
+  //   startDate: new Date("2023-04-20T00:00:00"),
+  //   endDate: new Date("2024-01-20T00:00:00"),
+  //   taskName: "23.04 (Lunar Lobster)",
+  //   status: "INTERIM_RELEASE",
+  // },
   {
     startDate: new Date("2023-10-20T00:00:00"),
     endDate: new Date("2024-07-20T00:00:00"),
     taskName: "23.10 (Mantic Minotaur)",
     status: "INTERIM_RELEASE",
   },
+  {
+    startDate: new Date("2024-04-25T00:00:00"),
+    endDate: new Date("2034-04-25T00:00:00"),
```

Comment:
Are these dates confirmed by the stakeholder?

Reply:
These are standard days for releases. Whom should I confirm with?

Reply:
Dates confirmed with Oliver